daemon/stats/collector_unix_test.go

```diff
@@ -0,0 +1,46 @@
+package stats // import "github.com/docker/docker/daemon/stats"
```

Comment:
The `_unix` suffix is not one that Golang itself recognises, so this needs to have a build tag to exclude it for windows;

```suggestion
// +build !windows

package stats // import "github.com/docker/docker/daemon/stats"
```

Reply:
Good catch, thanks! Adding it now.